homeworks/Vladimir.Karbivnychiy_VladimirSkarb/8-components-templates/pictures-slider/src/app/components/image-gallery/gallery.component.ts

```diff
@@ -0,0 +1,47 @@
+import {Component, OnInit} from '@angular/core';
+import {dataImg} from '../../core/data';
+import {DataInterface} from '../../interfaces/dataInterface';
+
+@Component({
+  selector: 'app-gallery',
+  templateUrl: './gallery.component.html',
+  styleUrls: ['./gallery.component.css']
+})
+export class GalleryComponent implements OnInit{
+
+  constructor() {
+  }
+
+  list: Array<DataInterface> = dataImg;
+  fullImage?: DataInterface;
+
+  ngOnInit(): void {
+    this.fullImage = this.list[0];
+  }
+
+  showItem(id: string): void {
+    this.fullImage = this.list.find((image: DataInterface) => image.id === id);
+  }
+
+  nextEl(): void {
```

Comment:
I assume that this method **show** next element. Each function in JS should be named from the verb:
- `showList`
- `hideItem`
- `matchString`
- `replaceAll`

Fix your `nextEl` and `prevEl` names